src/main/scala/firrtl/stage/FirrtlAnnotations.scala

```diff
@@ -64,6 +64,41 @@ object FirrtlFileAnnotation extends HasShellOptions {
 
 }
 
+/** Read a directory of FIRRTL files or ProtoBufs
+  *  - set with `-I/--input-directory`
+  * @param file input filename
+  */
+case class FirrtlDirectoryAnnotation(dir: String) extends NoTargetAnnotation with CircuitOption {
+
+  def toCircuit(info: Parser.InfoMode): FirrtlCircuitAnnotation = {
+    val circuit =
+      try {
+        proto.FromProto.fromDirectory(dir)
```

Comment:
this seems to be only reading protobuf, should the `FirrtlDirectoryAnnotation` be named `FirrtlProtobufDirectoryAnnoatation`?

Reply:
In the case of the analogous `FirrtlFileAnnotation`, we support both `.pb` and `.fir` with the same annotation, I think we should do the analogous thing here. Even if we're not implementing reading directories of `.fir` files in this PR, we should keep the name of the annotation general.

Reply:
ah, ok, so maybe the scaladoc at the top should be updated to reflect the "FIRRTL files" is "TODO"